setup.py

```diff
@@ -3,66 +3,46 @@
 import os
 import sys
 
-import jwt
-
-from setuptools import setup
-from setuptools.command.test import test
-
-
-class PyTest(test):
-    user_options = [('pytest-args=', 'a', "Arguments to pass to py.test")]
-
-    def initialize_options(self):
-        test.initialize_options(self)
-        self.pytest_args = []
-
-    def finalize_options(self):
-        test.finalize_options(self)
-        self.test_args = []
-        self.test_suite = True
-
-    def run_tests(self):
-        # import here, cause outside the eggs aren't loaded
-        import pytest
-        errno = pytest.main(['.'] + self.pytest_args)
-        sys.exit(errno)
+from setuptools import find_packages, setup
 
+__version__ = None
+with open('jwt/__init__.py') as fp:
+    _locals = {}
+    for line in fp:
+        if line.startswith('__version__'):
+            exec(line, None, _locals)
+            __version__ = _locals['__version__']
+            break
```

Comment:
Oh I see when it might actually fail. Let's do this instead.

```python
def get_version(package):
    """
    Return package version as listed in `__version__` in `init.py`.
    """
    init_py = open(os.path.join(package, '__init__.py')).read()
    return re.search("__version__ = ['\"]([^'\"]+)['\"]", init_py).group(1)

version = get_version('jwt')
```

That's what we use [here](https://github.com/tomchristie/django-rest-framework/blob/master/setup.py).